1.24.0/rockcraft.yaml

```diff
@@ -0,0 +1,53 @@
+# Rock for [install-cni](https://github.com/istio/istio/blob/master/cni/deployments/kubernetes/Dockerfile.install-cni).
+# This image's purpose is described [here](https://github.com/istio/istio/tree/master/cni#overview)
+
+name: install-cni
+summary: Istio's install-cni in a rock
+description: "Installs Istio's cni plugin"
+version: "1.24.0"
+base: ubuntu@24.04
+build-base: ubuntu@24.04
+services:
+  install-cni:
+    command: "/usr/local/bin/install-cni"
+    override: replace
+    startup: enabled
+    working-dir: /opt/cni/bin
+
+platforms:
+  amd64:
+
+environment:
+  PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/cni/bin
```

Comment:
A comment for why this is needed would be nice.

Reply:
tbh I have no idea why its needed.  I've added a comment to link to where it is done [upstream](https://github.com/istio/istio/blob/master/cni/deployments/kubernetes/Dockerfile.install-cni#L24), but I'm not sure why

Reply:
A wild guess I think is because `/opt/cni/bin` isn't going to be part of the default `PATH` and the `istio-cni` binary is in this directory.